### PR TITLE
qtgui: toggle label&icon on plot play/stop button

### PR DESCRIPTION
**Pull request by jasonuher**
Makes two changes:
  - First, toggle the label of the 'stop' button depending on the state in
    the same way the context menu does
  - Second, adds a visual 'play' or 'stop' icon based on the action

Signed-off-by: Jason Uher <jason.uher@jhuapl.edu>

### CODE REVIEW

**Comment by jasonuher** [comment]
I know this wasn't asked for, but I needed to do it for something else and thought you might want it.
I think I hit all the new guidelines, except the rebase since there were no changes to master I think.

**Comment by jasonuher** [comment]
hm, I'm must not have run clang correctly on my machine somehow.  let me try again

**Comment by mbr0wn** [comment]
> I know this wasn't asked for, but I needed to do it for something else and thought you might want it.
> I think I hit all the new guidelines, except the rebase since there were no changes to master I think.

There's hardly a better reason to submit something! Thanks! Can you please post a screenshot?

**Review by mbr0wn — APPROVED** [review]

**Comment by jasonuher** [comment]
sure thing
qt_gui_screenshots
